model/pom.xml

```diff
@@ -81,6 +81,21 @@
         <jooq.input-schema>PUBLIC</jooq.input-schema>
       </properties>
     </profile>
+
+    <profile>
+      <id>docker</id>
+      <properties>
+        <skipResetDatabase>true</skipResetDatabase>
+        <db.driver>org.h2.Driver</db.driver>
+        <db.create-url>jdbc:h2:/tmp/h2_data/keywhizdb_test</db.create-url>
```

Comment:
this should probably be `jdbc:h2:/data/keywhizdb_development` but I'm not sure if it's even being used by anything, so not a big deal.